### PR TITLE
Feature/issue 111

### PR DESCRIPTION
**Pull request by Urszuja**
Medication Category and test for it ready.

### CODE REVIEW

File: backend/src/models/MedicationCategory.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  PATCHES_COLOR,
+} from '../constants/pillColors';
+
+import { PILLS_ICON, SYRUP_ICON, INHALER_ICON, INJECTION_ICON, DROPS_ICON, PATCHES_ICON } from '../constants/medsIcons';
+
+const { Schema } = mongoose;
+
+const MedicationCategorySchema = new Schema({
+  name: String,
+  unit: { type: String, enum: ['ml', 'pcs'], default: 'pcs' },
```
Comment (Nefariusek) [inline backend/src/models/MedicationCategory.js:17]: Let's keep 'ml', 'pcs' in constants

File: backend/src/models/MedicationCategory.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  INJECTION_COLOR,
+  DROPS_COLOR,
+  PATCHES_COLOR,
+} from '../constants/pillColors';
+
+import { PILLS_ICON, SYRUP_ICON, INHALER_ICON, INJECTION_ICON, DROPS_ICON, PATCHES_ICON } from '../constants/medsIcons';
+
+const { Schema } = mongoose;
+
+const MedicationCategorySchema = new Schema({
```
Comment (Nefariusek) [inline backend/src/models/MedicationCategory.js:15]: Keep the convention that schema variable is camelcase (medicationCategorySchema)

**Review by Nefariusek — CHANGES_REQUESTED** [review]

File: backend/src/models/MedicationCategory.js (hunk 1 above)
Comment (Urszuja) [reply backend/src/models/MedicationCategory.js]: done

File: backend/src/models/MedicationCategory.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+  INJECTION_ICON,
+  DROPS_ICON,
+  PATCHES_ICON,
+} from '../constants/MedicationCategory/medsIcons';
+
+const { Schema } = mongoose;
+
+const medicationCategorySchema = new Schema({
+  name: String,
+  unit: { type: String, enum: [VOLUME_UNIT, QUANTITY_UNIT], default: 'pcs' },
```
Comment (Nefariusek) [inline backend/src/models/MedicationCategory.js:24]: 'pcs' as constant

**Review by Nefariusek — CHANGES_REQUESTED** [review]

File: backend/src/models/MedicationCategory.js (hunk 3 above)
Comment (Urszuja) [reply backend/src/models/MedicationCategory.js]: ok

**Comment by Urszuja** [comment]
changes introduced

**Pull request merged**